src/executor/userOpMonitor.ts

```diff
@@ -283,7 +268,7 @@ export class UserOpMonitor {
         const { executor, bundle } = submittedBundle
         const { userOps, entryPoint } = bundle
 
-        this.pendingBundles.delete(executor.address)
+        this.stopTrackingBundle(submittedBundle)
```

Comment:
stopTrackingBundle should be updated to delete based on bundle.uid, currently it is using the old implementation (executor address)

Reply:
good catch